how-to/business-logic/how-to-change-user-role.md

```diff
@@ -0,0 +1,72 @@
+---
+title: How To Change User Role Programatically
+description: An example on how to change the user role using cloud code
+type: how-to
+page_title: How To Apply Filtering Criteria Before Returning Data in the {{site.bs-bl}} Layer?
+slug: bl-apply-filter
+position:
+tags: backendservices, business-logic, cloud-code, user-role, cloud-functions
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Business Logic</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Cloud Functions</td>
+  </tr>
+</table>
+
+## Description
+
+You need to change/set a user's role programatically based on a certain logic (without going to Telerik Platform portal).

```

Comment:
Replace Telerik Platform with {{site.tp}}, this applies to all product and service name mentions in the article.